### ISSUE TITLE
Symfony 4-5 - doesn't work

### ISSUE THREAD

**Issue by seyfer** [dszczer/ideolog-symfony-format#1]
Hello

Thank you for the readme. I have tried it, it doesn't work with the latest Symfony 4. Shall it be adjusted?

Connected issue: https://github.com/JetBrains/ideolog/issues/68

**Comment by nicwortel** [comment]
I got it to work with the following pattern:

```regex
^\[(\d{4}-\d{2}-\d{2}\s+\d{2}:\d{2}:\d{2})\]\s+(\w+).(\w+):\s+(.*)$
```

![afbeelding](https://user-images.githubusercontent.com/1055691/73062682-39478f00-3e9d-11ea-8948-1b3ad69829b2.png)

**Comment by dszczer** [comment]
Fixed and merged to master with [ecebcf6](https://github.com/dszczer/ideolog-symfony-format/commit/ecebcf68d7132e345ce23ae3f3967aefadf9f27a) commit.
